### PR TITLE
codeintel: set db field in UploadHandler to fix panic when checking site-admin

### PR DESCRIPTION
**Pull request by Strum355**
cc @camdencheek

### CODE REVIEW

**Review by camdencheek — APPROVED** [review]
Whoops, my bad. Thanks for the fix 👍

**Review by efritz — APPROVED** [review]

File: enterprise/cmd/frontend/internal/codeintel/httpapi/upload_handler_test.go [hunk 1]
```diff
@@ -421,6 +428,100 @@ func TestHandleEnqueueMultipartFinalizeIncompleteUpload(t *testing.T) {
...
+		}
+
+		h := &UploadHandler{
+			db:          db,
+			dbStore:     mockDBStore,
+			uploadStore: mockUploadStore,
+			validators: map[string]func(context.Context, *http.Request, string) (int, error){
+				"github": func(context.Context, *http.Request, string) (int, error) {
+					if user.name != "owning-user" {
+						return http.StatusUnauthorized, errors.New("sample text import cycle")
```
Comment (efritz) [inline enterprise/cmd/frontend/internal/codeintel/httpapi/upload_handler_test.go:511]: lol

**Review by efritz — APPROVED** [review]
🎉 

Thank you for the test 😍

File: enterprise/cmd/frontend/internal/codeintel/httpapi/upload_handler_test.go (hunk 1 above)
Comment (Strum355) [reply enterprise/cmd/frontend/internal/codeintel/httpapi/upload_handler_test.go:511]: couldnt import `httpapi` which exposes the errors used :shrug:

**Pull request merged**